public/docs/_examples/animations/ts/app/hero-list-twoway.component.ts

```diff
@@ -0,0 +1,50 @@
+// #docregion
+// #docregion imports
+import {
+ Component,
+ Input,
+ trigger,
+ state,
+ style,
+ transition,
+ animate
+} from '@angular/core';
+// #enddocregion imports
+import { Hero, Heroes } from './hero.service';
+
+@Component({
+ moduleId: module.id,
+ selector: 'hero-list-twoway',
+ // #docregion template
+ template: `
+   <ul>
+     <li *ngFor="let hero of heroes.get()"
+         @heroState="hero.state"
+         (click)="hero.toggleActivation()">
+       {{hero.name}}
+     </li>
+   </ul>
+ `,
+ // #enddocregion template
+ styleUrls: ['hero-list.component.css'],
+ // #docregion animationdef
+ animations: [
+   trigger('heroState', [
+     state('inactive', style({
+       backgroundColor: '#eee',
+       transform: 'scale(1)'
+     })),
+     state('active',   style({
+       backgroundColor: '#cfd8dc',
+       transform: 'scale(1.1)'
+     })),
+     // #docregion transitions
+     transition('inactive <=> active', animate('100ms ease-out'))
```

Comment:
Comment here that transitions supports **two-way** state changes in a single definition.